include/libswiftnav/signal.h

```diff
@@ -19,15 +20,35 @@
 #define NUM_SATS_SBAS 19
 #define NUM_SATS (NUM_SATS_GPS + NUM_SATS_SBAS)
 
+#define CODES_ON_GPS_SAT 2
+#define CODES_ON_SBAS_SAT 1
+
+#define NUM_CODES_GPS (NUM_SATS_GPS * CODES_ON_GPS_SAT)
```

Comment:
I think NUM_CODES_GPS should be the number of different types of codes on GPS. So L1CA + L2CM + L2CL = 3 ?

NUM_SIGNALS_GPS should be NUM_SATS_GPS * NUM_CODES_GPS